model/docs/getting-started.qmd

```diff
@@ -98,6 +98,13 @@ latent_infections = Infections()
 observation_process = PoissonObservation()
 ```
 
+Notice that `Infections0` is an instance of the `DistributionalRV(RandomVariable)` class; therefore, the following could also be used:
```

Comment:
I would prefer to remove the `latent.I0` and `latent.InfectHospRate` classes altogether and have this be the only way to do things.

Reply:
Sure. Confirm this, @dylanhmorris?

Reply:
Happy for it to be removed for now. A couple thoughts:

1. This comes back to the "Wrapper classes for readability y/n?" question we were discussing earlier. In the case of simple scalar parameters I'm not sure the wrapper class is needed.
2. In the future, one might imagine wanting infrastructure for dealing, e.g., with static versus time-varying IHRs. In that scenario, an/some `InfectHospRate` class(es) might be the appropriate solution. But as those would _not_ all be simple wrappers of `DistributionalRV`, again I think removing the class for now makes sense.

Reply:
1. Yes, in this case, it is beyond readability. All objects passed to `Model` need a `sample` method. `numpyro.distributions.Distribution` does not. So we have the wrapper class that allows you to do so.
2. Absolutely, in the future we could have more complex methods, that's what I had in mind when I first created these classes, but we can revisit when the time comes.

Reply:
Let's make this a separate issue so we can land this PR.

Reply:
Addressed by c95d031 and 7ef0e55.